admin/admin-model.php

```diff
@@ -43,11 +43,9 @@ public function add_language( $args ) {
 		wp_update_term( (int) $r['term_id'], 'language', array( 'term_group' => (int) $args['term_group'] ) ); // can't set the term group directly in wp_insert_term
 
 		// The other language taxonomies.
-		foreach ( $this->translatable_objects->get_secondary_translatable_objects() as $object ) {
-			wp_insert_term( $args['name'], $object->get_tax_language(), array( 'slug' => 'pll_' . $args['slug'] ) );
-		}
+		$this->update_secondary_language_terms( $args['slug'], $args['name'] );
 
-		$this->clean_languages_cache(); // Update the languages list now !
+		$this->clean_languages_cache(); // Update the languages list now!
```

Comment:
I thought we had already removed this line. But it looks like it's not the case.

It should not be needed anymaore. It was previously useful to access incomplete `PLL_Language` object in `PLL_MO` and in action `pll_add_language` but we want to prevent this now.

Reply:
Removed in [b055000](https://github.com/polylang/polylang/pull/1176/commits/b055000b93861cacd840af1ef29c28fe0bcf46e2).